persistent/interfaces.py

```diff
@@ -23,7 +23,7 @@
     from persistent.cPersistence import UPTODATE
     from persistent.cPersistence import CHANGED
     from persistent.cPersistence import STICKY
-except ImportError: #pragma NO COVER
+except ImportError: # pragma: no cover
```

Comment:
I was likely reacting to the fact that all the other changes to this module were janitorial:  I still want to see them gone, and would prefer to see this fix backed out, too.  (Note that `tox -e coverage` already says we are at 100% coverage, even with the comment as is).